values/aws.yml

```
@@ -48,8 +48,8 @@ cvmfs:
 
 postgresql:
   persistence:
-    storageClass: nfs
+    storageClass: eb
```

No longer matches storage class name above
```suggestion
    storageClass: block_storage
```

---

Right...